### DISCUSSION TITLE
Cannot see "blocks" or "blocked by" properties on app.plane.so

### DISCUSSION THREAD

**Discussion by ianhinder** [makeplane/plane#3386] [General]
Hi,

I am trying out Plane for the first time, and have created a test project on app.plane.so.  I would like to mark an issue as blocking another issue, but despite the documentation saying there should be "blocks" and "blocked by" properties in the issue, I can't see these.

<img width="406" alt="image" src="https://github.com/makeplane/plane/assets/213274/55c9c2b4-8e4a-477b-ab16-c6680da518c0">

Is this functionality available, and do I have to do something special to enable it?

**Comment by srinivaspendem** [comment]
@ianhinder it's a bug, we will be fixing it soon. But it's not visible today in the peek overview mode, once you open the issue in full screen you can see it

**Comment by ianhinder** [comment]
Thanks.  This is a bit of a UI problem. I saw the little icons for full screen in the top of the sidebar, but it never occurred to me that this would make additional functionality visible; I just assumed it would make the existing window take the full screen.  I had never seen this "full screen" view of the issue from clicking around.  There was a very short view which appeared when I clicked on "Edit issue", which is different to the "side peek", and different again to the full screen view.  Maybe these could be unified in a more straightforward way, because at the moment it's a bit confusing.  Apologies if this has already been discussed!   A minimal UI fix would be to add a button at the bottom of the side peek to "expand details" or something, which would then lead to the current full screen view.
